pipelines/package_names.go

```diff
@@ -78,7 +78,7 @@ func TarOptsFromFileName(filename string) TarFileOpts {
 		// arm-7 should become arm/v7
 		arch = strings.Join([]string{archv[0], archv[1]}, "/")
 	}
-	edition := ""
+	edition := "oss"
```

Comment:
On second thought I'm not ok with this change. The edition is not "oss" the edition is nothing. I think this is reflected in the test that's failing

Reply:
I really want us to get away from calling "Grafana" "Grafana OSS". It's just "Grafana".

Reply:
Done on d68c40685d00989c4f83364949af6ae6d0c084fe